core/templates/dev/head/pages/thanks.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{% block subtitle %}
```

Comment:
Also noticed this is wrong in the Donate page. Perhaps fix that too or file a bug for it?